osu.iOS/osu.iOS.csproj

```diff
@@ -116,5 +116,9 @@
       <Visible>false</Visible>
     </ImageAsset>
   </ItemGroup>
+  <ItemGroup>
+  <!-- Used for obtaining battery info. -->
```

Comment:
Probably doesn't need to be stated. It's not hard to track which code is using a package by right-clicking on the package.

```suggestion
```